### ISSUE TITLE
[Azure Identity] Readme content and sample code of Azure Identity is incorrect

### ISSUE THREAD

**Issue by wantedfast** [Azure/azure-sdk-for-net#11330]
1.
Section [Link](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#chaining-credentials)
![image](https://user-images.githubusercontent.com/25993729/79318457-981b2280-7f39-11ea-9f3d-a49f72a8dbc3.png)

Comments: Both Classes name inaccurate in currently SDK, code snippet change as be below:
```
using Azure.Identity;
using Azure.Messaging.EventHubs.Producer;

var managedCredential = new ManagedIdentityCredential(clientId);
var certCredential = new ClientCertificateCredential(tenantId, clientId, certificate);
 
// authenticate using managed identity if it is available otherwise use certificate auth
var credential = new ChainedTokenCredential(managedCredential, certCredential);

var eventHubProducerClient = new EventHubProducerClient("myeventhub.eventhubs.windows.net", "myhubpath", credential);    
```

2.
Section [Link](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#authenticating-a-service-principal-with-a-certificate)
![image](https://user-images.githubusercontent.com/25993729/79318941-44f59f80-7f3a-11ea-9c38-95ee0a292219.png)

Comments: Suggest add `using System.Security.Cryptography.X509Certificates;` since X509Certificate2 class depends on this namespace. And it would help a new learner can run this sample code smoothly.
Change `CertificateCredential` to `ClientCertificateCredential` 

3.
Section [Link](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#authenticating-a-user-with-the-default-browser)
![image](https://user-images.githubusercontent.com/25993729/79319245-a4ec4600-7f3a-11ea-8b2c-0739f282e348.png)

Comments: Change it to `InteractiveBrowserCredential`

4.
Section [Link](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#authenticating-a-user-with-the-device-code-flow)
![image](https://user-images.githubusercontent.com/25993729/79319589-19bf8000-7f3b-11ea-8142-cafeae8bdcb1.png)

Comments: Comments and code are incorrect. Change it as below:
```
 // Create a secret client using the DeviceCodeCredential
var client = new SecretClient(new Uri("https://myvault.azure.vaults.net/"), new DeviceCodeCredential((code, cancellationToken) => PrintDeviceCode(code), clientId));
```

**Comment by jongio** [comment]
Please PR.  Thanks

**Comment by jsquire** [comment]
Fixed by #11360.     The PR is currently in an approved state and waiting for merge.

**Comment by wantedfast** [comment]
Close this issue since PR merged
